modules/nw-rfhe-introduction.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="nw-rfhe-introduction_{context}"]
+= How Redfish hardware events work
+
+OpenShift Container Platform now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. These hardware events are delivered over a reliable low-latency transport channel based on Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10~20 millisecond range.
+
+The hardware event proxy provides a publish-subscribe service for the hardware events, where multiple applications can use REST APIs to subscribe and consume the events.
+The hardware event proxy supports hardware compliant with Redfish OpenAPI v1.8 or higher.
+
+[NOTE]
+====
+This feature is supported for single node OpenShift clusters only.
+====
+
+[id="rfhe-elements_{context}"]
+== Redfish hardware events data flow
+
+The following figure illustrates the basic Redfish hardware events data flow:
+
+.Redfish hardware events data flow
+image::211_OpenShift_Redfish_dataflow_0222.png[Redfish hardware events data flow]
+
+=== Redfish hardware
+Hardware that is compatible with the Redfish standard and exposes hardware-related events through Redfish.
+
+=== Redfish event
+Redfish events are triggered by conditions such as excess temperatures, hardware outages such as server, router, or fan failure, and updates on network elements that have or could go wrong.
+A typical use case for Redfish hardware events is tracking faults in Edge devices such as DUs, but they can be useful in tracking a wide variety of hardware networking environments.
+
+=== REST APIs
+REST APIs provide a publish-subscribe model for the exhange of information between producing and subscribing applications. A REST API establishes the content required by the subscribing application (the call) and the content produced by the hardware event proxy (the response).
+
+=== Operator-managed pod
+The Operator makes use of custom resources to manage the pod containing the hardware event proxy and its components.
+
+=== Hardware event proxy
+A container created to handle Redfish events. At startup, the hardware event proxy queries the Redfish API and downloads all the Message Registries, including custom registries. The hardware event proxy then begins to receive subscribed events from the Redfish hardware.
+
+=== Cloud Native Event
+Cloud Native Events (CNE) is a Rest-API specification for defining the format of event data. https://github.com/redhat-cne/spec/blob/main/event_spec.md
```

Comment:
Check if we can link to this repo. If yes, please write a full sentence referencing it.